docs/docs/platform/authentication/sign-in-methods/3-phone-number.mdx

```diff
@@ -7,22 +7,53 @@ image: /img/og/platform/sign-in-with-phone-number-sms.png
 
 Follow this guide to sign in users with a phone number (SMS).
 
+## Setup
+
+You need a [Twillio account](https://www.twilio.com/try-twilio) to use this feature.
+
+Enable the Phone Number (SMS) sign-in method in the Nhost dashboard under **Users** -> **Login settings** -> **Passwordless SMS**.
+
+You need to insert the following settings in the Nhost dashboard from Twilio:
+
+- Account SID
+- Auth Token
+- Messaging Service SID (or a Twilio phone number)
+
+<video width="99%" autoPlay muted loop controls="true" style={{marginBottom: '15px'}}>
+  <source src="/videos/enable-sms-sign-in.mp4" type="video/mp4" />
+</video>
+
+
 ## Sign In
 
 To sign in users with a phone number is a two-step process:
 
 1. Send a one-time password (OTP) to the user's phone number.
-2. The user uses the OTP to sign in
+2. The user uses the OTP to sign in.
 
 ```js
 // Step 1: Send OTP to the user's phone number
 await nhost.auth.signIn({
-  phoneNumber: '0011233213123'
+  phoneNumber: '+11233213123'
 })
 
 // Step 2: Sign in user using their phone number and OTP
 await nhost.auth.signIn({
-  phoneNumber: '0011233213123'
+  phoneNumber: '+11233213123'
+  // highlight-next-line
   otp: '123456',
 })
 ```
+
+The first time a user signs in using a phone number, the user is created. That means you don't need to sign up the user before signin in the user.
+
+:::info
+
+It's recommended to start all phone numbers with `+` (and not `00`) to follow the [E.164 formatting standard](https://en.wikipedia.org/wiki/E.164).
+
+:::
+
+
+## Other SMS Providers
+
+We only support Twillio for now. If you want another SMS provider, please create an issue on [Hasura Auth on GitHub](https://github.com/nhost/hasura-auth).
```

Comment:
Although it's correct it should end up in a hasura-auth issue, I'd redirect to our main repo to generate more traffic